zipline/finance/commission.py

```diff
@@ -108,3 +111,135 @@ def calculate(self, transaction):
         """
         cost_per_share = transaction.price * self.cost
         return cost_per_share, abs(transaction.amount) * cost_per_share
+
+
+class PerShareWithOptions(object):
+    """
+    Calculates a commission for a transaction based on a per
+    share cost with an optional minimum cost per trade.
+    Applies different set of fees to equities and underliers.
+    """
+
+    def __init__(self, equity_cost=DEFAULT_PER_SHARE_COST,
+                 equity_min_trade_cost=DEFAULT_MINIMUM_COST_PER_TRADE,
+                 option_costs=None,
+                 option_min_trade_cost=DEFAULT_MINIMUM_COST_PER_TRADE,
+                 equity_sids=None, option_sids=None):
+        """
```

Comment:
for zipline we have started using numpy style docstrings. This would change this slightly to look like:

```
Parameters
----------
equity_cost : float, optional
    Cost of an equity trade per-share.
equity_min_trade_cost : float, optional
    The minimum trade cost for equities.
...
```

This should also be merged with the class docstring.
See below for some comments about the names of these parameters

A full guide can be found here: https://github.com/numpy/numpy/blob/master/doc/HOWTO_DOCUMENT.rst.txt